src/app/pages/vehicle.cpp

```diff
@@ -102,11 +102,14 @@ VehiclePage::VehiclePage(QWidget *parent) : QTabWidget(parent)
     for (auto port : QSerialPortInfo::availablePorts())
         this->serial_devices.append(port.systemLocation());
 
+    this->can_devices.append("unloader");
```

Comment:
maybe call this "null" or something similar? just to not give the impression that this is somehow a plugin

Reply:
Renamed to "No Interface" which I think might be the clearest wording.